### DISCUSSION TITLE
No executor found for ip:port when starting a Spark App.

### DISCUSSION THREAD

**Discussion by paulpaul1076** [stackabletech/spark-k8s-operator#352] [Q&A]
When I start Spark apps, there's usually an info message like this:

```
2024-02-08T14:10:43,424 INFO [dispatcher-CoarseGrainedScheduler] org.apache.spark.scheduler.cluster.k8s.KubernetesClusterSchedulerBackend$KubernetesDriverEndpoint - No executor found for 11.43.33.49:33352
```

Is it normal?

**Comment by razvan** [comment]
Hard to say. The [source](https://github.com/apache/spark/blob/ce5ddad990373636e94071e7cef2f31021add07b/resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/k8s/KubernetesClusterSchedulerBackend.scala#L353) suggests it's expected. Are there any  warnings or errors related to that IP in the logs ?

**Reply by paulpaul1076** [reply]
No, I think it only happens in the beginning for the most part, when executors are just starting. But sometimes I saw it in the middle of jobs.
